### PR TITLE
Modifiy the padding-bottom for h1 - Closes #859

### LINKED ISSUE #859: General UI update
The UI needs to be polished up, particular elements can have better spacing/padding before the final release.

The feedback from UI designer:

- [x] Move “Home” a little bit to the top, increasing the margin-bottom.
- [x] Have more distance in between the filters on the right, so they don’t collide. Let’s think also if we can move them inside the container.
- [x] Improve the padding in between H1+subtext and the other content
- [x] On the main explorer, get rid of the map if is useless and adds no-information and give more room to stats on the left side.
- [x] Align “Last Block by” and “Last block” Vertically.
- [x] Align “ID” in the table vertically with the H1 above it.
_- [ ] Let’s make sure that we have always some spacing in between, if we keep “Latest transactions” and “Latest blocks” in one raw._

• Top Nav

- [x] Let’s make sure that logo and links are aligned with the container below, so they are part of the same grid.

• Transactions

- [x] As of now there is too much space in between H1 and container, let’s  remove some of this extra space, to have the list closer.
- [x] Pagination needs some more work -> #858 

• Blocks

- [x] Same as above.

• Delegate Monitor

- [x] Spacing improvements
- [x] Let’s align the data below vertically so for e.g. “Last block” should be aligned vertically with “Best Forger” below.
- [x] We can also improve a little bit the padding for the tabs, especially for numbers inside the rectangles, that are too close to the edges.
- [x] We can also think about improving a bit the styling of the stats below inside the rectangles, to make it look more appealing :slightly_smiling_face:

• Network

- [x] Spacing improvements on the top part
- [x] Let’s improve the organisation of the stats above, to have the same space horizontally.
- [x] Statistics: let’s have the same padding-left and right as of the other tables.

Functional issues:
- [x] Pagination fails when number of pages is greater that 0 and  less than 5

### PR DESCRIPTION
**Pull request by youxiberlin**
### What was the problem?
There was too much space in between H1 and container, 

### How did I fix it?
Modified the padding value of h1.

### How to test it?
Checked on a browser.

### Review checklist

* The PR solves #859 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the
	[commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: src/assets/styles/tableMobile.css [hunk 1]
```diff
@@ -74,4 +75,4 @@
 	#table-mobile td:before {
 		content: attr(data-title);
 	}
-}
+}
```
Comment (MichalTuleja) [inline src/assets/styles/tableMobile.css]: The last line should be an empty line

File: src/assets/styles/tableMobile.css [hunk 2]
```diff
@@ -32,7 +33,7 @@
 	#table-mobile td, 
 	#table-mobile tr { 
 		display: block;
-		padding: 10px 0;
+		/* padding: 10px 0; */
```
Comment (MichalTuleja) [inline src/assets/styles/tableMobile.css]: Do we really need to remove that spacing entirely? 5px or less might look better

File: src/assets/styles/tableMobile.css (hunk 1 above)
Comment (youxiberlin) [reply src/assets/styles/tableMobile.css]: Added an empty line.

File: src/assets/styles/tableMobile.css (hunk 2 above)
Comment (youxiberlin) [reply src/assets/styles/tableMobile.css]: Agree. Made it 5 px.

**Pull request merged**